build/config/ios/rules.gni

```diff
@@ -131,7 +135,7 @@ template("ios_app") {
   copy_gen_target_name = target_name + "_copy"
   copy(copy_gen_target_name) {
     sources = [
-      "$root_build_dir/Info.plist",
+      "$root_build_dir/plist/$app_name/Info.plist",
```

Comment:
root_build_dir seems wrong.  That's like out/ios_sim_Debug, I think you want $target_gen_dir or something?